lxqt-base/liblxqt/liblxqt-0.11.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-utils
+
+DESCRIPTION="Common base library for the LXQt desktop environment"
+HOMEPAGE="http://lxqt.org/"
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lxde/${PN}.git"
+else
+	SRC_URI="https://github.com/lxde/${PN}/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+fi
+
+LICENSE="GPL-2 LGPL-2.1+"
+SLOT="0"
+
+RDEPEND=">=dev-libs/libqtxdg-1.3.0:=
+	dev-qt/qtcore:5=
+	dev-qt/qtdbus:5=
+	dev-qt/qtgui:5=
+	dev-qt/qtwidgets:5=
+	dev-qt/qtx11extras:5=
+	dev-qt/qtxml:5=
+	kde-frameworks/kwindowsystem:5=[X]
+	x11-libs/libXScrnSaver"
+DEPEND="${RDEPEND}
+	>=dev-util/lxqt-build-tools-0.4.0
+	dev-qt/linguist-tools:5="
+
+PATCHES=(
+	"${FILESDIR}"/${P}-build-{1,2,3}.patch
+)
+
+pkg_pretend() {
+	if [[ ${MERGE_TYPE} != binary ]]; then
+		tc-is-gcc && [[ $(gcc-version) < 4.8 ]] && \
+		die 'The active compiler needs to be gcc 4.8 (or newer)'
+	fi
+}
```

Comment:
remove this, noone in this day and age still uses GCC 4.7

Reply:
Same